perception/lidar_centerpoint/src/node.cpp

```diff
@@ -55,6 +55,7 @@ LidarCenterPointNode::LidarCenterPointNode(const rclcpp::NodeOptions & node_opti
   const std::string head_onnx_path = this->declare_parameter<std::string>("head_onnx_path");
   const std::string head_engine_path = this->declare_parameter<std::string>("head_engine_path");
   class_names_ = this->declare_parameter<std::vector<std::string>>("class_names");
+  has_variance_ = this->declare_parameter("has_variance", false);
```

Comment:
Same comment with above

```suggestion
  has_variance_ = this->declare_parameter<bool>("has_variance");
```